### PR TITLE
Feature/new deploy

### LINKED ISSUE #40: Fix gh-pages routing Issues
# Bug
Browser Router is not allowing access to routes/pages not accessed through the index page and also not allowing direct navigation to specific pages (e.g. `deployeddomain/join/1` is returning a 404 error although it works when run locally).

## Potential Starting Point
- This [article](https://gmfuster.medium.com/deploying-a-react-app-to-github-pages-24c3e5485589) suggests changing the router type (tried this first with no success)
- The Dev Community [post](https://dev.to/dilinar/comment/1n837) suggests a redirection approach with an index.html file (see spa-github-pages link)
- Github community [post](https://github.com/orgs/community/discussions/50269)
- Into using a different deployment service (netlifly? render?)

### PR DESCRIPTION
**Pull request by EthanDuvall**
# Brain Defrost FE

## Summary 
This pr udates our code to be deployed to netlify due to github pages not working well with single page applications 
## Related Issues
closes #40
## Changes Made
changed our package.json to use our updated link as well as our index and got rid of the base name 
## Checklist
- [x] The code follows the project's coding standards.
- [ ] Unit tests have been added or updated to cover the changes.
- [ ] Documentation has been updated to reflect the changes (if applicable).
- [x] The code compiles without errors.
- [ ] All new and existing tests pass.
- [x] The pull request has been reviewed by at least one other contributor.

### CODE REVIEW

**Comment by tednaphil** [comment]
@EthanDuvall I'm not able to run the app on `localhost` from this branch - I'm being shown a blank page w/ errors in the console (see below). We can't merge these changes yet since the tests can't access the app from `localhost` and are failing.

Can we touch base tomorrow during standup to try to get it going and figure out what's happening? Thank you!
Screenshot 2024-05-30 at 9 34 52 PM

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "brain-defrost-fe",
   "version": "0.1.0",
-  "homepage": "https://Brain-Defrost.github.io/Brain-Defrost_FE",
+  "homepage": "https://https://brain-defrost.netlify.app",
```
Comment (tednaphil) [inline package.json:4]: There are two `https://`s  - assuming one will need to be deleted

**Review by tednaphil — CHANGES_REQUESTED** [review]
See the note on the `package.json` file! I don't know what else we'll need to troubleshoot to get the app working on `localhost` again, but I'm hopeful that'll be some progress!

**Comment by EthanDuvall** [comment]
The commits i just pushed up should have fixed the test issue as well as it not running on local host all i did was change the hompage link to "." which allows it to run on botht he deployed link as well as our other local hoast let me know if thers any hcanges youd like !

File: cypress/e2e/App.cy.ts [hunk 2]
```diff
@@ -1,7 +1,7 @@
 describe('Brain Defrost User Stories', () => {
   beforeEach(() => {
     // cy.viewport(1025, 700)
-    cy.visit('http://localhost:3000/Brain-Defrost_FE')
+    cy.visit('http://localhost:3000')
```
Comment (tednaphil) [inline cypress/e2e/App.cy.ts:4]: Thanks for getting the tests updated too!

**Review by tednaphil — APPROVED** [review]
Great work getting the deployment up on netlify!

**Pull request merged**